store/cachekv/store.go

```diff
@@ -312,6 +392,7 @@ func (store *Store) dirtyItems(start, end []byte) {
 
 	// Now find the values within the domain
 	//  [start, end)
+	store.gasMeter.ConsumeGas(store.gasConfig.ReadCostFlat, types.GasReadCostFlatDesc)
```

Comment:
The algorithm in this method is `O(N)`, don't sounds right to assign a constant cost.